packages/portal/src/components/item/ItemLanguageSelector.vue

```diff
@@ -109,20 +76,29 @@
       locales
     ],
     props: {
+      /**
+       * Translation request error
+       */
       fromTranslationError: {
         type: Boolean,
         default: false
       },
+      /**
+       * Language the item is translated to
+       */
       translationLanguage: {
         type: String,
         default: null
+      },
+      behindLogin: {
```

Comment:
is this only used to allow the setup in the styleguide?

Reply:
Yes. Now that we moved the login to the options this is no longer needed.